addons/hr/views/hr_views.xml

```diff
@@ -246,11 +246,6 @@
                                         <field name="permit_no"/>
                                         <field name="visa_expire"/>
                                     </group>
-                                    <group string="Education">
```

Comment:
Out of scope change?

Reply:
yes added by error during test fix

already removed meanwhile